mappings/c8bbde6c4d181fd2da84b7dea3db5c956604b984754af770cb5e58e64257415245.json

```diff
@@ -42,8 +42,8 @@
         ]
     },
     "description": {
-        "sequenceNumber": 0,
-        "value": "Cardano Native Token of The bWare Group. BWARE is aiming to revolutionise the economic systems currently in use and create a better experience for every one.",
+        "sequenceNumber": 1,
+        "value": "Aiming to create a better experience for everyone!",
```

Comment:
Hi @Digitalbwt,

Once you update the values, you should sign them again. Here it looks like signatures are unchanged, hence validation failing.

Please update the PR

Reply:
Hello @VladislavKudrin 

Sorry for the mistakes, seems like I missed the push last time. Now it passed the validation, hope everything is fine now. Thank you!